docs/api-specs/reporting-api-v1.yaml

```diff
@@ -84,13 +97,21 @@ paths:
                 $ref: '#/components/schemas/ReportCalculation'
         401:
           description: If the access token does not grant permission to this Report
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
         404:
           description: If report does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
 
     post:
       security:
         - development:
-          - reports_write
+          - reporting_write
       tags:
         - report
       summary: Trigger a new report calculation run.
```

Comment:
Suggested update to the "description" part below (can't comment on that here in the PR ...):

> ~(CURRENTLY UNDER DEVELOPMENT) optionally receive status updates via a webhook if that has been set up for the authorized client~

> As an alternative to triggering a single report calculation, you can subscribe to receive a notification event (which includes an automatically created calculationId) whenever the report's results change (see /webhook/{webhookId}/subscribe/report).